### ISSUE TITLE
Windows issue - Could not initialize class org.bytedeco.javacpp.avutil

### ISSUE THREAD

**Issue by AlexDBlack** [deeplearning4j/DataVec#130]
I thought there was an issue open for this already (I've seen this before, it's not new), but apparently not...
I'm guessing this is windows specific.

```
java.lang.NoClassDefFoundError: Could not initialize class org.bytedeco.javacpp.avutil

	at org.bytedeco.javacv.FFmpegFrameFilter.startUnsafe(FFmpegFrameFilter.java:158)
	at org.bytedeco.javacv.FFmpegFrameFilter.start(FFmpegFrameFilter.java:154)
	at org.datavec.image.transform.FilterImageTransform.<init>(FilterImageTransform.java:61)
	at org.datavec.image.transform.TestImageTransform.testFilterImageTransform(TestImageTransform.java:135)
```

**Comment by saudet** [comment]
Right, it's not used for anything important right now, so not a blocker for the release.

**Comment by saudet** [comment]
Fixed in 0.7.2.
